tests/crypto/Kconfig

```diff
@@ -0,0 +1,10 @@
+config TV_ASSERT_USER_OVERRIDE
+	bool "Override test vector asserts"
+    default n
```

Comment:
indentation

Reply:
config removed in 7aee7a9 , this is set in prj.conf